site/en/adminGuide/monitor/metrics_dashboard.md

```diff
@@ -0,0 +1,163 @@
+| Msg Rows Consum Rate  | The number of rows of streaming messages consumed per minute by each data node within the past two minutes.  | ```  sum(increase(milvus_datanode_msg_rows_count{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])/2) by (msg_type, pod, node_id)  ```  | `milvus_datanode_msg_rows_count`  | The number of rows of streaming messages consumed.     <br/>    Currently, streaming messages counted by data node only include insertion and deletion messages.   |
+| Flush Data Size Rate  | The size of each flushed message recorded per minute by each data node within the past two minutes.  | ```  sum(increase(milvus_datanode_flushed_data_size{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])/2) by (msg_type, pod, node_id)  ```  | `milvus_datanode_flushed_data_size`  | The size of each flushed message.    <br/>    Currently, streaming messages counted by data node only include insertion and deletion messages.   |
+| Consumer Num  | The number of consumers created on each data node.     | ```  sum(milvus_datanode_consumer_num{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (pod, node_id)  ```  | `milvus_datanode_consumer_num`  | The number of consumers created on each data node.     <br/>    Each flowgraph corresponds to a consumer.  |
+| Producer Num  | The number of producers created on each data node.     | ```  sum(milvus_datanode_producer_num{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (pod, node_id)  ```  | `milvus_datanode_producer_num`  | The number of consumers created on each data node.     <br/>    Each shard in a collection corresponds to a delta channel producer and a timetick channel producer.  |
+| Sync Time  | The average, maximum, and minimum number of epoch time synced by each data node in all physical topics.  | ```  avg(milvus_datanode_sync_epoch_time{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (pod, node_id)  max(milvus_datanode_sync_epoch_time{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (pod, node_id)  min(milvus_datanode_sync_epoch_time{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (pod, node_id)  ```  | `milvus_datanode_sync_epoch_time`  | The epoch time (Unix time, the milliseconds passed ever since January 1, 1970.) of each physical topic on a data node.     |
```

Comment:
better delete the point in the bracket "...the milliseconds passed ever since January 1, 1970)"

Reply:
This is provided in the original copy.